src/modules/involvment-api.js

```diff
@@ -0,0 +1,49 @@
+const involId = 'fn8dJlnUFl6vA8TtKscd';
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps';
+const likeUrl = `${baseUrl}/${involId}/likes/`;
+
+/*
+const getLikes = () => fetch(likeUrl)
+  .then((result) => result.json());
+
+export default getLikes;
+*/
```

Comment:
- [ ] Please remove useless comment in the code because it make your code look messy, kindly remove it